cmd/auth/login.go

```diff
@@ -15,18 +15,37 @@ import (
 	"github.com/spf13/cobra"
 )
 
-func configureHost(ctx context.Context, persistentAuth *auth.PersistentAuth, args []string, argIndex int) error {
-	if len(args) > argIndex {
-		persistentAuth.Host = args[argIndex]
-		return nil
+func promptForProfile(ctx context.Context, dv string) (string, error) {
+	if !cmdio.IsInTTY(ctx) {
+		return "", errors.New("the command is being run in a non-interactive environment, please specify a profile using --profile")
 	}
 
-	host, err := promptForHost(ctx)
+	prompt := cmdio.Prompt(ctx)
+	prompt.Label = "Databricks Profile Name"
```

Comment:
```suggestion
	prompt.Label = "Databricks profile name [DEFAULT]"
```

Reply:
And I wonder, should we even ask for this? Isn't it an advanced use case to have multiple profiles? Can't you use `--profile` if you're an advanced user?

Reply:
What behaviour do you suggest? Write to the default profile if it does not exist. If it does, then prompt for a profile name?

Reply:
A default is supplied so folks can enter through this and modify only if they want to.

Reply:
I modified the prompt to include "DEFAULT" as the default.

Reply:
Thanks; at the very least we should just indicate in the prompt that there is a DEFAULT as seen above.

What is the use case for always asking for a profile, though? That still seems like it mostly caters to advanced users.

Reply:
> Thanks; at the very least we should just indicate in the prompt that there is a DEFAULT as seen above

Yeah, that's there:
```
➜  cli git:(fix/auth-login-host) cli auth login                                                       
✔ Databricks profile name: DEFAULT█
```

I tend to agree that we don't always need to ask for the profile here. Let me do a followup PR for that though, the scope for this one is already expanding more than I intended.